src/Ziggy.php

```diff
@@ -22,12 +20,7 @@ public function __construct(string $group = null, string $url = null)
         $this->group = $group;
 
         $this->baseUrl = rtrim($url ?? config('app.url', url('/')), '/');
-
-        tap(parse_url($this->baseUrl), function ($url) {
-            $this->baseProtocol = $url['scheme'] ?? 'http';
-            $this->baseDomain = $url['host'] ?? '';
-            $this->basePort = $url['port'] ?? null;
-        });
+        $this->basePort = parse_url($this->baseUrl)['port'] ?? null;
```

Comment:
Just wanna make sure, same as the other PR, that this gets updated to `$this->url` in whatever order these PRs are merged. Not necessarily here, probably in #338, but don't wanna forget.

Reply:
👍🏻